PyInstaller/utils/win32/versioninfo.py

```diff
@@ -101,7 +102,7 @@ def pefile_read_version(filename):
 
 
 def getRaw(o):
-    return str(buffer(o))
```

Comment:
Note that this previously worked because `str(buffer(o))` converts from `unicode` to `str` without any encoding operation - it simply takes the bytes from the internal storage of the `unicode`, which is UTF-16 LE encoded *as an implementation detail of CPython for Windows* and wraps them in a `str`. It's basically a static cast.